node/actors/control_api.go

```diff
@@ -199,44 +291,107 @@ func (api *ControlAPI) handleInfo(m *nats.Msg) {
 		return
 	}
 	// Ask the agent supervisor for a list of all the workloads from all of its children
-	response, err := api.self.Ask(ctx, agentSuper, req)
+	response, err := api.self.Ask(ctx, agentSuper, new(actorproto.GetNodeInfo))
 	if err != nil {
-		api.logger.Error("Failed to get list of running workloads from agent supervisor", slog.Any("error", err))
-		respondEnvelope(m, InfoResponseType, 500, nil, fmt.Sprintf("failed to get list of running workloads: %s", err))
+		api.logger.Error("Failed to get node info", slog.Any("error", err))
```

Comment:
This actually made more sense the other way. We're asking the agent supervisor for the list of agents and their running workloads. We use that information as _part_ of the node info, but the agent supervisor shouldn't be handling a "node info" query

Reply:
yup...not even sure why I changed it